### PR TITLE
Prepare twitter card

### PR DESCRIPTION
**Pull request by goetas**
Following up on @Jean85 suggestion done over slack..

### CODE REVIEW

**Comment by goetas** [comment]
Do we have a PHP-FIG logo? possibly in SVG or a large PNG? 

Something as this:

image


but bigger..?

File: source/_layouts/base.twig [hunk 1]
```diff
@@ -25,6 +25,24 @@
 
     <link rel="preconnect" href="https://ajax.googleapis.com" pr="1.0" />
     <link rel="preconnect" href="https://ssl.google-analytics.com" pr="0.8" />
+
+    <meta property="og:url" content="{{ ('https://www.php-fig.org/' ~ page.url) }}" />
```
Comment (goetas) [inline source/_layouts/base.twig]: do we have some parameters for the deploy url?

Comment (goetas) [reply source/_layouts/base.twig]: Twitter wants the image URL to be absolute.... this should be something provided by plafrorm sh i guess...

Comment (Jean85) [reply source/_layouts/base.twig]: I see that you removed it. Probably the URL itself of the page being linked is enough, we don't need to care about permanent URLs due to the fact that the site is static.

Comment (Jean85) [reply source/_layouts/base.twig]: Nope, I tested it, the URL has to be absolute for sure. Worst case, we could use the official domain.

Comment (Jean85) [reply source/_layouts/base.twig]: PSH gives us the variables in the build here: https://docs.platform.sh/development/variables.html#platformsh-provided-variables

I don't know if it's worth the hassle for something that we do not change basically ever...

File: .platform.app.yaml [hunk 2]
```diff
@@ -29,6 +29,9 @@ hooks:
     ruby -v
     curl -sS https://platform.sh/cli/installer | php
     gem install bundler
+
+    sed -i "s/host:.*/host: https:\/\/www---pr-263-pugnbma-ovkfaqrdltr5s.us.platform.sh/" app/config/sculpin_site.yml
```
Comment (goetas) [inline .platform.app.yaml]: this line is temporary and it is only for testing this PR. should be removed before merging

**Comment by goetas** [comment]
Twitter is back operational, here is the preview: 
image


Ready for review!

**Comment by goetas** [comment]
@php-fig/secretaries

**Comment by Jean85** [comment]
I'm not sure about the preview with that kind of format, with the logo on top. In the past, @michaelcullum told me that it wasn't "standard", and in fact event stickers were produced with the "normal" layout:

**Comment by goetas** [comment]
Hmm...but that was the one i did before... and your suggestion was to use the one is on github... where the "check sign" is on top, not on the left

**Comment by goetas** [comment]
The first version I've used was Logo_Light

**Comment by Jean85** [comment]
Sorry for letting you do the job twice then :(

I'll revert to the previous banner and leave everything else as is, so I can publish it together with #265. We can always revert/amend later :+1:

File: source/_layouts/post.twig [hunk 3]
```diff
@@ -18,3 +18,11 @@
         </div>
     {% endfor %}
 {% endblock %}
+
+{% block description_block %}
+    {%- if page.description -%}
+        {{ page.description |replace("\n", " ") }}
```
Comment (Jean85) [inline source/_layouts/post.twig]: Crap my bad, this line was wrong (see fix e9db1fd930e22fca851f1f7a328fec6c9b1b8bc4) and it broke a few posts.

**Comment by rob006** [comment]
@Jean85 Also https://github.com/php-fig/www.php-fig.org/pull/263#discussion_r330086026

**Comment by Jean85** [comment]
Thanks @rob006! Cleaned in a85f4ca

**Comment by goetas** [comment]
Ive commented here about https://github.com/php-fig/www.php-fig.org/pull/263#discussion_r330086026 the test script, but Ive missed https://github.com/php-fig/www.php-fig.org/commit/e9db1fd930e22fca851f1f7a328fec6c9b1b8bc4 

Thanks for fixing!

**Pull request merged**